src/shogun/features/SubsetStack.cpp

```diff
@@ -63,6 +66,7 @@ void CSubsetStack::remove_all_subsets()
 		m_active_subsets_stack->delete_element(i);
 
 	SG_UNREF(m_active_subset);
+	m_active_subset = nullptr;
```

Comment:
this is needed because:

- `~DenseFeatures()` calls `SubsetStack->remove_all_subsets()`, that unrefs `m_active_subset`
- `~Features()` unrefs `m_subset_stack` triggering its dtor `~CSubsetStack()` that unrefs `m_active_subset` a second time

so setting it to null avoids the second unref